copy/progress_bars_test.go

```diff
@@ -0,0 +1,24 @@
+package copy
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vbauerster/mpb/v8/decor"
+)
+
+func TestCustomPartialBlobDecorFunc(t *testing.T) {
```

Comment:
(This works just fine. In Go, it is idiomatic to have table-driven tests, compare e.g. `TestUpdatedBlobInfoFromUpload`.)